src/main/kotlin/app/revanced/cli/utils/signature/SignatureDownloader.kt

```diff
@@ -0,0 +1,22 @@
+package app.revanced.cli.utils.signature
+
+import java.io.File
+import java.net.URL
+import java.nio.file.Files
+import java.nio.file.Paths
+
+object SignatureDownloader {
+
+    fun download(out: String) {
+        val url =
+            URL("https://raw.githubusercontent.com/ReVancedTeam/revanced-signatures/main/signatures/youtube.signatures.json")
```

Comment:
This is the wrong branch for now (until PRs are merged), but you can keep it like this.

Reply:
i'll take a look

Reply:
epic